<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><title>An S4 class to represent a set of PGS Catalog Polygenic Scores — scores-class • quincunx</title><!-- favicons --><link rel="icon" type="image/png" sizes="16x16" href="../favicon-16x16.png"><link rel="icon" type="image/png" sizes="32x32" href="../favicon-32x32.png"><link rel="apple-touch-icon" type="image/png" sizes="180x180" href="../apple-touch-icon.png"><link rel="apple-touch-icon" type="image/png" sizes="120x120" href="../apple-touch-icon-120x120.png"><link rel="apple-touch-icon" type="image/png" sizes="76x76" href="../apple-touch-icon-76x76.png"><link rel="apple-touch-icon" type="image/png" sizes="60x60" href="../apple-touch-icon-60x60.png"><script src="../deps/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><link href="../deps/bootstrap-5.3.1/bootstrap.min.css" rel="stylesheet"><script src="../deps/bootstrap-5.3.1/bootstrap.bundle.min.js"></script><link href="../deps/Nunito-0.4.9/font.css" rel="stylesheet"><link href="../deps/Inconsolata-0.4.9/font.css" rel="stylesheet"><link href="../deps/Source_Sans_Pro-0.4.9/font.css" rel="stylesheet"><link href="../deps/font-awesome-6.5.2/css/all.min.css" rel="stylesheet"><link href="../deps/font-awesome-6.5.2/css/v4-shims.min.css" rel="stylesheet"><script src="../deps/headroom-0.11.0/headroom.min.js"></script><script src="../deps/headroom-0.11.0/jQuery.headroom.min.js"></script><script src="../deps/bootstrap-toc-1.0.1/bootstrap-toc.min.js"></script><script src="../deps/clipboard.js-2.0.11/clipboard.min.js"></script><script src="../deps/search-1.0.0/autocomplete.jquery.min.js"></script><script src="../deps/search-1.0.0/fuse.min.js"></script><script src="../deps/search-1.0.0/mark.min.js"></script><!-- pkgdown --><script src="../pkgdown.js"></script><meta property="og:title" content="An S4 class to represent a set of PGS Catalog Polygenic Scores — scores-class"><meta name="description" content="The scores object consists of six tables (slots) that combined form a
relational database of a subset of PGS Catalog polygenic scores. Each score
is an observation (row) in the scores table (the first table)."><meta property="og:description" content="The scores object consists of six tables (slots) that combined form a
relational database of a subset of PGS Catalog polygenic scores. Each score
is an observation (row) in the scores table (the first table)."><meta property="og:image" content="https://rmagno.eu/quincunx/logo.svg"></head><body>
    <a href="#main" class="visually-hidden-focusable">Skip to contents</a>


    <nav class="navbar navbar-expand-lg fixed-top bg-none" data-bs-theme="light" aria-label="Site navigation"><div class="container">

    <a class="navbar-brand me-2" href="../index.html">quincunx</a>

    <small class="nav-text text-muted me-auto" data-bs-toggle="tooltip" data-bs-placement="bottom" title="">0.1.8</small>


    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div id="navbar" class="collapse navbar-collapse ms-3">
      <ul class="navbar-nav me-auto"><li class="active nav-item"><a class="nav-link" href="../reference/index.html">Reference</a></li>
<li class="nav-item dropdown">
  <button class="nav-link dropdown-toggle" type="button" id="dropdown-articles" data-bs-toggle="dropdown" aria-expanded="false" aria-haspopup="true">Articles</button>
  <ul class="dropdown-menu" aria-labelledby="dropdown-articles"><li><hr class="dropdown-divider"></li>
    <li><h6 class="dropdown-header" data-toc-skip>PGS Catalog Concepts</h6></li>
    <li><a class="dropdown-item" href="../articles/identifiers.html">Identifiers</a></li>
    <li><a class="dropdown-item" href="../articles/pgs-construction-process.html">PGS Construction Process and Stages</a></li>
    <li><a class="dropdown-item" href="../articles/cohorts-samples-sample-sets.html">Cohorts, Samples and Sample Sets</a></li>
    <li><hr class="dropdown-divider"></li>
    <li><h6 class="dropdown-header" data-toc-skip>Getting PGS Catalog Data</h6></li>
    <li><a class="dropdown-item" href="../articles/getting-pgs-scores.html">Getting PGS Scores</a></li>
    <li><a class="dropdown-item" href="../articles/getting-pgs-publications.html">Getting PGS Publications</a></li>
    <li><a class="dropdown-item" href="../articles/getting-pgs-sample-sets.html">Getting PGS Sample Sets</a></li>
    <li><a class="dropdown-item" href="../articles/getting-pgs-performance-metrics.html">Getting PGS Performance Metrics</a></li>
    <li><a class="dropdown-item" href="../articles/getting-traits.html">Getting Traits</a></li>
    <li><a class="dropdown-item" href="../articles/getting-cohorts.html">Getting Cohorts</a></li>
    <li><a class="dropdown-item" href="../articles/getting-releases.html">PGS Catalog Releases</a></li>
    <li><a class="dropdown-item" href="../articles/pgs-scoring-file.html">PGS scoring file</a></li>
    <li><hr class="dropdown-divider"></li>
    <li><h6 class="dropdown-header" data-toc-skip>Wrangling PGS Catalog Data</h6></li>
    <li><a class="dropdown-item" href="../articles/pgs-cat-ent-subsetting.html">PGS Catalog Entity Subsetting</a></li>
    <li><hr class="dropdown-divider"></li>
    <li><h6 class="dropdown-header" data-toc-skip>Case Studies</h6></li>
    <li><a class="dropdown-item" href="../articles/risk-strat-brs-patients.html">Risk of Brugada syndrome patients</a></li>
    <li><a class="dropdown-item" href="../articles/pgs-scores-mavaddat.html">Exploring PGS scores by Mavaddat et al. (2018)</a></li>
    <li><hr class="dropdown-divider"></li>
    <li><a class="dropdown-item" href="../articles/index.html">More articles...</a></li>
  </ul></li>
<li class="nav-item"><a class="nav-link" href="../news/index.html">Changelog</a></li>
      </ul><ul class="navbar-nav"><li class="nav-item"><form class="form-inline" role="search">
 <input class="form-control" type="search" name="search-input" id="search-input" autocomplete="off" aria-label="Search site" placeholder="Search for" data-search-index="../search.json"></form></li>
<li class="nav-item"><a class="external-link nav-link" href="https://github.com/ramiromagno/quincunx/" aria-label="GitHub"><span class="fa fab fa-github fa-lg"></span></a></li>
      </ul></div>


  </div>
</nav><div class="container template-reference-topic">
<div class="row">
  <main id="main" class="col-md-9"><div class="page-header">
      <img src="../logo.svg" class="logo" alt=""><h1>An S4 class to represent a set of PGS Catalog Polygenic Scores</h1>
      <small class="dont-index">Source: <a href="https://github.com/ramiromagno/quincunx/blob/master/R/class-scores.R" class="external-link"><code>R/class-scores.R</code></a></small>
      <div class="d-none name"><code>scores-class.Rd</code></div>
    </div>

    <div class="ref-description section level2">
    <p>The scores object consists of six tables (slots) that combined form a
relational database of a subset of PGS Catalog polygenic scores. Each score
is an observation (row) in the <code>scores</code> table (the first table).</p>
    </div>


    <div class="section level2">
    <h2 id="slots">Slots<a class="anchor" aria-label="anchor" href="#slots"></a></h2>


<dl><dt><code>scores</code></dt>
<dd><p>A table of polygenic scores. Each polygenic score (row) is
uniquely identified by the <code>pgs_id</code> column. Columns:</p><dl><dt>pgs_id</dt>
<dd><p>Polygenic Score (PGS) identifier. Example: <code>"PGS000001"</code>.</p></dd>

<dt>pgs_name</dt>
<dd><p>This may be the name that the authors describe the PGS with
in the source publication, or a name that a curator of the PGS Catalog has
assigned to identify the score during the curation process (before a PGS
identifier has been given). Example: <code>PRS77_BC</code>.</p></dd>

<dt>scoring_file</dt>
<dd><p>URL to the scoring file on the PGS FTP server. Example:
<code>"http://ftp.ebi.ac.uk/pub/databases/spot/pgs/scores/PGS000001/ScoringFiles/PGS000001.txt.gz"</code>.</p></dd>

<dt>matches_publication</dt>
<dd><p>Indicate if the PGS data matches the published
polygenic score (<code>TRUE</code>). If not (<code>FALSE</code>), the authors have
provided an alternative polygenic for the Catalog and some other data, such
as performance metrics, may differ from the publication.</p></dd>

<dt>reported_trait</dt>
<dd><p>The author-reported trait that the PGS has been
developed to predict. Example: <code>"Breast Cancer"</code>.</p></dd>

<dt>trait_additional_description</dt>
<dd><p>Any additional description not captured
in the other columns. Example: <code>"Femoral neck BMD (g/cm2)"</code>.</p></dd>

<dt>pgs_method_name</dt>
<dd><p>The name or description of the method or computational
algorithm used to develop the PGS.</p></dd>

<dt>pgs_method_params</dt>
<dd><p>A description of the relevant inputs and parameters
relevant to the PGS development method/process.</p></dd>

<dt>n_variants</dt>
<dd><p>Number of variants used to calculate the PGS.</p></dd>

<dt>n_variants_interactions</dt>
<dd><p>Number of higher-order variant interactions
included in the PGS.</p></dd>

<dt>assembly</dt>
<dd><p>The version of the genome assembly that the variants present
in the PGS are associated with. Example: <code>GRCh37</code>.</p></dd>

<dt>license</dt>
<dd><p>The PGS Catalog distributes its data according to EBI's
standard Terms of Use. Some PGS have specific terms, licenses, or
restrictions (e.g. non-commercial use) that we highlight in this field, if
known.</p></dd>


</dl></dd>


<dt><code>publications</code></dt>
<dd><p>A table of publications. Each publication (row) is
uniquely identified by the <code>pgp_id</code> column. Columns:</p><dl><dt>pgs_id</dt>
<dd><p>Polygenic Score (PGS) identifier.</p></dd>

<dt>pgp_id</dt>
<dd><p>PGS Publication identifier. Example: <code>"PGP000001"</code>.</p></dd>

<dt>pubmed_id</dt>
<dd><p><a href="https://en.wikipedia.org/wiki/PubMed" class="external-link">PubMed</a>
identifier. Example: <code>"25855707"</code>.</p></dd>

<dt>publication_date</dt>
<dd><p>Publication date. Example: <code>"2020-09-28"</code>. Note
that the class of <code>publication_date</code> is <code><a href="https://rdrr.io/r/base/Dates.html" class="external-link">Date</a></code>.</p></dd>

<dt>publication</dt>
<dd><p>Abbreviated name of the journal. Example: <code>"Am J Hum
Genet"</code>.</p></dd>

<dt>title</dt>
<dd><p>Publication title.</p></dd>

<dt>author_fullname</dt>
<dd><p>First author of the publication. Example:
<code>'Mavaddat N'</code>.</p></dd>

<dt>doi</dt>
<dd><p>Digital Object Identifier (DOI). This variable is also curated to
allow unpublished work (e.g. preprints) to be added to the catalog. Example:
<code>"10.1093/jnci/djv036"</code>.</p></dd>


</dl></dd>


<dt><code>samples</code></dt>
<dd><p>A table of samples. Each sample (row) is uniquely identified by
the combination of values from the columns: <code>pgs_id</code> and
<code>sample_id</code>. Columns:</p><dl><dt>pgs_id</dt>
<dd><p>Polygenic score identifier. An identifier that starts with
<code>'PGS'</code> and is followed by six digits, e.g. <code>'PGS000001'</code>.</p></dd>

<dt>sample_id</dt>
<dd><p>Sample identifier. This is a surrogate key to identify each sample.</p></dd>

<dt>stage</dt>
<dd><p>Sample stage: either <code>"discovery"</code> or <code>"training"</code>.</p></dd>

<dt>sample_size</dt>
<dd><p>Number of individuals included in the sample.</p></dd>

<dt>sample_cases</dt>
<dd><p>Number of cases.</p></dd>

<dt>sample_controls</dt>
<dd><p>Number of controls.</p></dd>

<dt>sample_percent_male</dt>
<dd><p>Percentage of male participants.</p></dd>

<dt>phenotype_description</dt>
<dd><p>Detailed phenotype description.</p></dd>

<dt>ancestry_category</dt>
<dd><p>Author reported ancestry is mapped to the best matching
ancestry category from the NHGRI-EBI GWAS Catalog framework (see
<code><a href="ancestry_categories.html">ancestry_categories</a></code>) for possible values.</p></dd>

<dt>ancestry</dt>
<dd><p>A more detailed description of sample ancestry
that usually matches the most specific description described by the authors
(e.g. French, Chinese).</p></dd>

<dt>country</dt>
<dd><p>Author reported countries of recruitment (if available).</p></dd>

<dt>ancestry_additional_description</dt>
<dd><p>Any additional description not
captured in the other columns (e.g. founder or genetically isolated
populations, or further description of admixed samples).</p></dd>

<dt>study_id</dt>
<dd><p>Associated GWAS Catalog study accession identifier, e.g.,
<code>"GCST002735"</code>.</p></dd>

<dt>pubmed_id</dt>
<dd><p><a href="https://en.wikipedia.org/wiki/PubMed" class="external-link">PubMed</a>
identifier.</p></dd>

<dt>cohorts_additional_description</dt>
<dd><p>Any additional description about the
samples (e.g. sub-cohort information).</p></dd>


</dl></dd>


<dt><code>demographics</code></dt>
<dd><p>A table of sample demographics' variables. Each
demographics' variable (row) is uniquely identified by the combination of
values from the columns: <code>pgs_id</code>, <code>sample_id</code> and
<code>variable</code>. Columns:</p><dl><dt>pgs_id</dt>
<dd><p>Polygenic Score (PGS) identifier.</p></dd>

<dt>sample_id</dt>
<dd><p>Sample identifier. This is a surrogate identifier to
identify each sample.</p></dd>

<dt>variable</dt>
<dd><p>Demographics variable. Following columns report about the
indicated variable.</p></dd>

<dt>estimate_type</dt>
<dd><p>Type of statistical estimate for variable.</p></dd>

<dt>estimate</dt>
<dd><p>The variable's statistical value.</p></dd>

<dt>unit</dt>
<dd><p>Unit of the variable.</p></dd>

<dt>variability_type</dt>
<dd><p>Measure of statistical dispersion for variable, e.g.
standard error (se) or standard deviation (sd).</p></dd>

<dt>variability</dt>
<dd><p>The value of the measure of dispersion.</p></dd>

<dt>interval_type</dt>
<dd><p>Type of statistical interval for variable: range, iqr
(interquartile), ci (confidence interval).</p></dd>

<dt>interval_lower</dt>
<dd><p>Interval lower bound.</p></dd>

<dt>interval_upper</dt>
<dd><p>Interval upper bound.</p></dd>


</dl></dd>


<dt><code>cohorts</code></dt>
<dd><p>A table of cohorts. Each cohort (row) is uniquely identified by
the combination of values from the columns: <code>pgs_id</code>, <code>sample_id</code>
and <code>cohort_symbol</code>. Columns:</p><dl><dt>pgs_id</dt>
<dd><p>Polygenic Score (PGS) identifier.</p></dd>

<dt>sample_id</dt>
<dd><p>Sample identifier. This is a surrogate key to identify each sample.</p></dd>

<dt>cohort_symbol</dt>
<dd><p>Cohort symbol.</p></dd>

<dt>cohort_name</dt>
<dd><p>Cohort full name.</p></dd>


</dl></dd>


<dt><code>traits</code></dt>
<dd><p>A table of EFO traits. Each trait (row) is uniquely identified
by the combination of the columns <code>pgs_id</code> and <code>efo_id</code>. Columns:</p><dl><dt>pgs_id</dt>
<dd><p>Polygenic Score (PGS) identifier.</p></dd>

<dt>efo_id</dt>
<dd><p>An <a href="https://www.ebi.ac.uk/efo/" class="external-link">EFO</a> identifier.</p></dd>

<dt>trait</dt>
<dd><p>Trait name.</p></dd>

<dt>description</dt>
<dd><p>Detailed description of the trait from EFO.</p></dd>

<dt>url</dt>
<dd><p>External link to the EFO entry.</p></dd>


</dl></dd>


<dt><code>stages_tally</code></dt>
<dd><p>A table of sample sizes and number of samples sets at each stage.</p><dl><dt>pgs_id</dt>
<dd><p>Polygenic Score (PGS) identifier.</p></dd>

<dt>stage</dt>
<dd><p>Sample stage: either <code>"gwas"</code>, <code>"dev"</code> or <code>"eval"</code>.</p></dd>

<dt>sample_size</dt>
<dd><p>Sample size.</p></dd>

<dt>n_sample_sets</dt>
<dd><p>Number of sample sets (only meaningful for the evaluation stage <code>"eval"</code>)</p></dd>


</dl></dd>


<dt><code>ancestry_frequencies</code></dt>
<dd><p>This table describes the ancestry composition at each stage.</p><dl><dt>pgs_id</dt>
<dd><p>Polygenic Score (PGS) identifier.</p></dd>

<dt>stage</dt>
<dd><p>Sample stage: either <code>"gwas"</code>, <code>"dev"</code> or <code>"eval"</code>.</p></dd>

<dt>ancestry_class_symbol</dt>
<dd><p>Ancestry class symbol.</p></dd>

<dt>frequency</dt>
<dd><p>Ancestry fraction (percentage).</p></dd>


</dl></dd>


<dt><code>multi_ancestry_composition</code></dt>
<dd><p>A table of a breakdown of the ancestries included in multi-ancestries.</p><dl><dt>pgs_id</dt>
<dd><p>Polygenic Score (PGS) identifier.</p></dd>

<dt>stage</dt>
<dd><p>Sample stage: either <code>"gwas"</code>, <code>"dev"</code> or <code>"eval"</code>.</p></dd>

<dt>multi_ancestry_class_symbol</dt>
<dd><p>Multi-ancestry class symbol.</p></dd>

<dt>ancestry_class_symbol</dt>
<dd><p>Ancestry class symbol.</p></dd>


</dl></dd>


</dl></div>

  </main></div>


    <footer><div class="pkgdown-footer-left">
  <p>Developed by <a href="https://www.pattern.institute/team/rmagno/" class="external-link">Ramiro Magno</a>, <a href="http://iduarte.eu/" class="external-link">Isabel Duarte</a>, Ana-Teresa Maia, <a href="https://cintesis.eu/" class="external-link"><img src="https://rmagno.eu/quincunx/reference/figures/cintesis.svg" height="24" alt="CINTESIS logo"></a>, <a href="https://www.pattern.institute/" class="external-link"><img src="https://rmagno.eu/quincunx/reference/figures/pi-logotype-canonical-whole-v01.svg" height="24" alt="Pattern Institute logo"></a>.</p>
</div>

<div class="pkgdown-footer-right">

</div>

    </footer></div>





  </body></html>

