<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>Learning about published polygenic risk score(s) • quincunx</title>
<!-- favicons --><link rel="icon" type="image/png" sizes="16x16" href="../favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="../favicon-32x32.png">
<link rel="apple-touch-icon" type="image/png" sizes="180x180" href="../apple-touch-icon.png">
<link rel="apple-touch-icon" type="image/png" sizes="120x120" href="../apple-touch-icon-120x120.png">
<link rel="apple-touch-icon" type="image/png" sizes="76x76" href="../apple-touch-icon-76x76.png">
<link rel="apple-touch-icon" type="image/png" sizes="60x60" href="../apple-touch-icon-60x60.png">
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><link href="../extra.css" rel="stylesheet">
<meta property="og:title" content="Learning about published polygenic risk score(s)">
<meta property="og:description" content="quincunx">
<meta property="og:image" content="https://rmagno.eu/quincunx/logo.svg">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">
    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">quincunx</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">0.1.0</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../index.html">
    <span class="fas fa-home fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Articles
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../articles/identifiers.html">Identifiers</a>
    </li>
    <li>
      <a href="../articles/learning-about-a-pgs.html">Learning about published polygenic risk score(s)</a>
    </li>
    <li>
      <a href="../articles/pgs-scoring-file.html">PGS scoring file</a>
    </li>
    <li>
      <a href="../articles/releases.html">PGS Catalog Releases</a>
    </li>
  </ul>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right"></ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      

      </header><script src="learning-about-a-pgs_files/accessible-code-block-0.0.1/empty-anchor.js"></script><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1 data-toc-skip>Learning about published polygenic risk score(s)</h1>
            
      
      
      <div class="hidden name"><code>learning-about-a-pgs.Rmd</code></div>

    </div>

    
    
<div id="introduction" class="section level2">
<h2 class="hasAnchor">
<a href="#introduction" class="anchor"></a>Introduction</h2>
<p>Let us say you are a researcher studying breast cancer and that you are interested in screening and prevention of this disease. Now, imagine you have just recently noticed a new publication that claims to have developed a set of new polygenic risk scores that are both powerful and reliable predictors of breast cancer risk, e.g., <a href="https://doi.org/10.1016/j.ajhg.2018.11.002">Mavaddat et al. (2018)</a>.</p>
<p>Perhaps now you would like to investigate a bit more about these new polygenic risk scores to assess their potential application. You know that the performance of these scores is dependent on various aspects, such as study design, participant demographics, case definitions, and covariates adjusted for. In general, to access this information, you will have to carefully read the paper for these details. If it happens however that these scores have been already indexed and curated by the <a href="https://www.pgscatalog.org/">PGS Catalog</a> team then you might take advantage of using this free and open resource to quickly gather relevant data about these scores. And if you happen to be an R user, then you can use quincunx retrieval functions to fetch polygenic score metadata from the PGS Catalog REST API server. This is what we will be doing next.</p>
</div>
<div id="finding-polygenic-risk-scores-by-publication" class="section level2">
<h2 class="hasAnchor">
<a href="#finding-polygenic-risk-scores-by-publication" class="anchor"></a>Finding polygenic risk scores by publication</h2>
<div id="searching-for-publications-by-author-name" class="section level3">
<h3 class="hasAnchor">
<a href="#searching-for-publications-by-author-name" class="anchor"></a>Searching for publications by author name</h3>
<p>To check if we are fortunate enough to have the polygenic risk scores for breast cancer (<a href="https://doi.org/10.1016/j.ajhg.2018.11.002">Mavaddat et al. (2018)</a>) in the PGS Catalog, we start by loading the quincunx package and by searching for this publication in the PGS Catalog. To do this we will use the function <code><a href="../reference/get_publications.html">get_publications()</a></code> and start by searching by the first author’s last name, i.e. “Mavaddat”:</p>
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span><span class="op">(</span><span class="va">quincunx</span><span class="op">)</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; Attaching package: 'quincunx'</span>
<span class="co">#&gt; The following object is masked from 'package:dplyr':</span>
<span class="co">#&gt; </span>
<span class="co">#&gt;     n</span>

<span class="va">pub_by_mavaddat</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/get_publications.html">get_publications</a></span><span class="op">(</span>author <span class="op">=</span> <span class="st">'Mavaddat'</span><span class="op">)</span> <span class="co"># Not case sensitive,</span>
                                                         <span class="co"># so "mavaddat" would</span>
                                                         <span class="co"># have worked just fine.</span></code></pre></div>
<p>When you ask the PGS Catalog for publications, it returns an S4 <code>publications</code> object with two tables (slots): <code>publications</code> and <code>pgs_ids</code>. You can access these tables with the <code><a href="https://rdrr.io/r/base/slotOp.html">@</a></code> operator.</p>
<p>The <code>publications</code> table (data frame) contains all the publications added to the PGS Catalog that contain an author whose last name is “Mavaddat”:</p>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">pub_by_mavaddat</span><span class="op">@</span><span class="va">publications</span>
<span class="co">#&gt; # A tibble: 6 x 8</span>
<span class="co">#&gt;   pgp_id pubmed_id publication_date publication title author_fullname doi  </span>
<span class="co">#&gt;   &lt;chr&gt;  &lt;chr&gt;     &lt;date&gt;           &lt;chr&gt;       &lt;chr&gt; &lt;chr&gt;           &lt;chr&gt;</span>
<span class="co">#&gt; 1 PGP00… 25855707  2015-04-08       J Natl Can… Pred… Mavaddat N      10.1…</span>
<span class="co">#&gt; 2 PGP00… 30554720  2018-12-13       Am J Hum G… Poly… Mavaddat N      10.1…</span>
<span class="co">#&gt; 3 PGP00… 28376175  2017-07-01       J Natl Can… Eval… Kuchenbaecker … 10.1…</span>
<span class="co">#&gt; 4 PGP00… 33022221  2020-10-05       Am J Hum G… Brea… Kramer I        10.1…</span>
<span class="co">#&gt; 5 PGP00… 32737321  2020-07-31       Nat Commun  Euro… Ho WK           10.1…</span>
<span class="co">#&gt; 6 PGP00… 32665703  2020-07-15       Genet Med   Poly… Barnes DR       10.1…</span>
<span class="co">#&gt; # … with 1 more variable: authors &lt;chr&gt;</span></code></pre></div>
<p>Seemingly there are 6 publications indexed in the PGS Catalog. Each publication added to the PGS Catalog gets an unique identifier. This identifier is right in the first column (<code>pgp_id</code>) of the <code>publications</code> table. The number of publications can be obtained by using the function <code><a href="../reference/n.html">n()</a></code> on the S4 object <code>pub_by_mavaddat</code>, or by asking directly the number of rows of the <code>publications</code> table:</p>
<div class="sourceCode" id="cb3"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="../reference/n.html">n</a></span><span class="op">(</span><span class="va">pub_by_mavaddat</span><span class="op">)</span>
<span class="co">#&gt; [1] 6</span>
<span class="fu"><a href="https://rdrr.io/r/base/nrow.html">nrow</a></span><span class="op">(</span><span class="va">pub_by_mavaddat</span><span class="op">@</span><span class="va">publications</span><span class="op">)</span>
<span class="co">#&gt; [1] 6</span></code></pre></div>
<p>Moreover, we can see that Nasim Mavaddat is not the first author in all these publications because her name is not always present in the column <code>author_fullname</code> (which contains the name of the first author only). If you want to know all the authors you can access the column <code>authors</code> (contains a string of comma separated author names).</p>
<p>Now, to identify our publication of interest amongst these 6 publications, we can check, for example, the PubMed identifier of <a href="https://doi.org/10.1016/j.ajhg.2018.11.002">Mavaddat et al. (2018)</a> (column <code>pubmed_id</code>), or check other information such as journal name (column <code>publication</code>), the publication title (column <code>title</code>) or the publication date (column <code>publication_date</code>). The most unambiguous approach is to use the PubMed identifier. The PubMed identifier for <a href="https://doi.org/10.1016/j.ajhg.2018.11.002">Mavaddat et al. (2018)</a> is <code>"30554720"</code>. So the publication of interest corresponds to the one returned in row number 2 of the <code>publications</code> table:</p>
<div class="sourceCode" id="cb4"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">publication_row_index</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/which.html">which</a></span><span class="op">(</span><span class="va">pub_by_mavaddat</span><span class="op">@</span><span class="va">publications</span><span class="op">$</span><span class="va">pubmed_id</span> <span class="op">==</span> <span class="st">"30554720"</span><span class="op">)</span>
<span class="va">publication_row_index</span>
<span class="co">#&gt; [1] 2</span></code></pre></div>
<p>To focus now on this publication and filter out the others, we can subset the <code>publications</code> object <code>pub_by_mavaddat</code>, e.g., by position, and get a new object with both tables (<code>publications</code> and <code>pgs_ids</code>) filtered for only this publication:</p>
<div class="sourceCode" id="cb5"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">mavaddat2018</span> <span class="op">&lt;-</span> <span class="va">pub_by_mavaddat</span><span class="op">[</span><span class="va">publication_row_index</span><span class="op">]</span>
<span class="va">mavaddat2018</span>
<span class="co">#&gt; An object of class "publications"</span>
<span class="co">#&gt; Slot "publications":</span>
<span class="co">#&gt; # A tibble: 1 x 8</span>
<span class="co">#&gt;   pgp_id pubmed_id publication_date publication title author_fullname doi  </span>
<span class="co">#&gt;   &lt;chr&gt;  &lt;chr&gt;     &lt;date&gt;           &lt;chr&gt;       &lt;chr&gt; &lt;chr&gt;           &lt;chr&gt;</span>
<span class="co">#&gt; 1 PGP00… 30554720  2018-12-13       Am J Hum G… Poly… Mavaddat N      10.1…</span>
<span class="co">#&gt; # … with 1 more variable: authors &lt;chr&gt;</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; Slot "pgs_ids":</span>
<span class="co">#&gt; # A tibble: 15 x 3</span>
<span class="co">#&gt;    pgp_id    pgs_id    stage      </span>
<span class="co">#&gt;    &lt;chr&gt;     &lt;chr&gt;     &lt;chr&gt;      </span>
<span class="co">#&gt;  1 PGP000002 PGS000004 development</span>
<span class="co">#&gt;  2 PGP000002 PGS000005 development</span>
<span class="co">#&gt;  3 PGP000002 PGS000006 development</span>
<span class="co">#&gt;  4 PGP000002 PGS000007 development</span>
<span class="co">#&gt;  5 PGP000002 PGS000008 development</span>
<span class="co">#&gt;  6 PGP000002 PGS000009 development</span>
<span class="co">#&gt;  7 PGP000002 PGS000001 evaluation </span>
<span class="co">#&gt;  8 PGP000002 PGS000002 evaluation </span>
<span class="co">#&gt;  9 PGP000002 PGS000003 evaluation </span>
<span class="co">#&gt; 10 PGP000002 PGS000004 evaluation </span>
<span class="co">#&gt; 11 PGP000002 PGS000005 evaluation </span>
<span class="co">#&gt; 12 PGP000002 PGS000006 evaluation </span>
<span class="co">#&gt; 13 PGP000002 PGS000007 evaluation </span>
<span class="co">#&gt; 14 PGP000002 PGS000008 evaluation </span>
<span class="co">#&gt; 15 PGP000002 PGS000009 evaluation</span></code></pre></div>
<p>We see now that the <code>publications</code> table contains one entry only, and that the PGS Catalog identifier assigned to this publication is PGP000002:</p>
<div class="sourceCode" id="cb6"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">mavaddat2018</span><span class="op">@</span><span class="va">publications</span><span class="op">$</span><span class="va">pgp_id</span>
<span class="co">#&gt; [1] "PGP000002"</span></code></pre></div>
</div>
<div id="searching-for-publications-by-pubmed-id" class="section level3">
<h3 class="hasAnchor">
<a href="#searching-for-publications-by-pubmed-id" class="anchor"></a>Searching for publications by PubMed ID</h3>
<p>An alternatively, albeit more direct, route to get this publication by <a href="https://doi.org/10.1016/j.ajhg.2018.11.002">Mavaddat et al. (2018)</a> could have been to query for publications directly by the corresponding PubMed ID (30554720):</p>
<div class="sourceCode" id="cb7"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">pub_by_pmid_30554720</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/get_publications.html">get_publications</a></span><span class="op">(</span>pubmed_id <span class="op">=</span> <span class="st">'30554720'</span><span class="op">)</span>
<span class="va">pub_by_pmid_30554720</span>
<span class="co">#&gt; An object of class "publications"</span>
<span class="co">#&gt; Slot "publications":</span>
<span class="co">#&gt; # A tibble: 1 x 8</span>
<span class="co">#&gt;   pgp_id pubmed_id publication_date publication title author_fullname doi  </span>
<span class="co">#&gt;   &lt;chr&gt;  &lt;chr&gt;     &lt;date&gt;           &lt;chr&gt;       &lt;chr&gt; &lt;chr&gt;           &lt;chr&gt;</span>
<span class="co">#&gt; 1 PGP00… 30554720  2018-12-13       Am J Hum G… Poly… Mavaddat N      10.1…</span>
<span class="co">#&gt; # … with 1 more variable: authors &lt;chr&gt;</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; Slot "pgs_ids":</span>
<span class="co">#&gt; # A tibble: 15 x 3</span>
<span class="co">#&gt;    pgp_id    pgs_id    stage      </span>
<span class="co">#&gt;    &lt;chr&gt;     &lt;chr&gt;     &lt;chr&gt;      </span>
<span class="co">#&gt;  1 PGP000002 PGS000004 development</span>
<span class="co">#&gt;  2 PGP000002 PGS000005 development</span>
<span class="co">#&gt;  3 PGP000002 PGS000006 development</span>
<span class="co">#&gt;  4 PGP000002 PGS000007 development</span>
<span class="co">#&gt;  5 PGP000002 PGS000008 development</span>
<span class="co">#&gt;  6 PGP000002 PGS000009 development</span>
<span class="co">#&gt;  7 PGP000002 PGS000001 evaluation </span>
<span class="co">#&gt;  8 PGP000002 PGS000002 evaluation </span>
<span class="co">#&gt;  9 PGP000002 PGS000003 evaluation </span>
<span class="co">#&gt; 10 PGP000002 PGS000004 evaluation </span>
<span class="co">#&gt; 11 PGP000002 PGS000005 evaluation </span>
<span class="co">#&gt; 12 PGP000002 PGS000006 evaluation </span>
<span class="co">#&gt; 13 PGP000002 PGS000007 evaluation </span>
<span class="co">#&gt; 14 PGP000002 PGS000008 evaluation </span>
<span class="co">#&gt; 15 PGP000002 PGS000009 evaluation</span></code></pre></div>
</div>
</div>
<div id="from-publication-to-polygenic-risk-scores" class="section level2">
<h2 class="hasAnchor">
<a href="#from-publication-to-polygenic-risk-scores" class="anchor"></a>From publication to polygenic risk scores</h2>
<p>Now that we have found that our publication of interest exists in the PGS Catalog, whose identifier is PGP000002, it is time now to check which polygenic risk scores are annotated in the Catalog. Polygenic scores (PGS) in the PGS Catalog are indexed by an unique accession identifier of the form: “PGS000000” (“PGS” followed by six digits).</p>
<p>To get all PGS identifiers associated with Mavaddat’s publication we turn to the second table <code>pgs_ids</code> that maps publication identifiers (PGP) to PGS identifiers:</p>
<div class="sourceCode" id="cb8"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">pub_by_pmid_30554720</span><span class="op">@</span><span class="va">pgs_ids</span>
<span class="co">#&gt; # A tibble: 15 x 3</span>
<span class="co">#&gt;    pgp_id    pgs_id    stage      </span>
<span class="co">#&gt;    &lt;chr&gt;     &lt;chr&gt;     &lt;chr&gt;      </span>
<span class="co">#&gt;  1 PGP000002 PGS000004 development</span>
<span class="co">#&gt;  2 PGP000002 PGS000005 development</span>
<span class="co">#&gt;  3 PGP000002 PGS000006 development</span>
<span class="co">#&gt;  4 PGP000002 PGS000007 development</span>
<span class="co">#&gt;  5 PGP000002 PGS000008 development</span>
<span class="co">#&gt;  6 PGP000002 PGS000009 development</span>
<span class="co">#&gt;  7 PGP000002 PGS000001 evaluation </span>
<span class="co">#&gt;  8 PGP000002 PGS000002 evaluation </span>
<span class="co">#&gt;  9 PGP000002 PGS000003 evaluation </span>
<span class="co">#&gt; 10 PGP000002 PGS000004 evaluation </span>
<span class="co">#&gt; 11 PGP000002 PGS000005 evaluation </span>
<span class="co">#&gt; 12 PGP000002 PGS000006 evaluation </span>
<span class="co">#&gt; 13 PGP000002 PGS000007 evaluation </span>
<span class="co">#&gt; 14 PGP000002 PGS000008 evaluation </span>
<span class="co">#&gt; 15 PGP000002 PGS000009 evaluation</span></code></pre></div>
<p>There are 15 scores. To retrieve information about them, we use the quincunx function <code><a href="../reference/get_scores.html">get_scores()</a></code>:</p>
<div class="sourceCode" id="cb9"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">mavaddat2018_scores</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/get_scores.html">get_scores</a></span><span class="op">(</span>pgs_id <span class="op">=</span> <span class="va">pub_by_pmid_30554720</span><span class="op">@</span><span class="va">pgs_ids</span><span class="op">$</span><span class="va">pgs_id</span><span class="op">)</span>
<span class="fu">slotNames</span><span class="op">(</span><span class="va">mavaddat2018_scores</span><span class="op">)</span>
<span class="co">#&gt; [1] "scores"       "publications" "samples"      "demographics" "cohorts"     </span>
<span class="co">#&gt; [6] "traits"</span></code></pre></div>
<p>The returned object is an S4 object <code>scores</code> which contains six tables (slots): scores, publications, samples, demographics, cohorts, traits.</p>
</div>
<div id="polygenic-scores-metadata" class="section level2">
<h2 class="hasAnchor">
<a href="#polygenic-scores-metadata" class="anchor"></a>Polygenic Scores Metadata</h2>
<div id="scores-table" class="section level3">
<h3 class="hasAnchor">
<a href="#scores-table" class="anchor"></a>Scores table</h3>
<p>The S4 scores object <code>mavaddat2018_scores</code> starts with a table named <code>scores</code> that lists each score in one row each. All scores are identified by the PGS identifier (column <code>pgs_id</code>). In addition, scores can have a name (column <code>pgs_name</code>). This may be the name that the authors describe the PGS with in the source publication, or a name that a curator of the PGS Catalog has assigned to identify the score during the curation process.</p>
<div class="sourceCode" id="cb10"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">mavaddat2018_scores</span><span class="op">@</span><span class="va">scores</span><span class="op">[</span><span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span><span class="op">(</span><span class="st">'pgs_id'</span>, <span class="st">'pgs_name'</span><span class="op">)</span><span class="op">]</span>
<span class="co">#&gt; # A tibble: 15 x 2</span>
<span class="co">#&gt;    pgs_id    pgs_name     </span>
<span class="co">#&gt;    &lt;chr&gt;     &lt;chr&gt;        </span>
<span class="co">#&gt;  1 PGS000004 PRS313_BC    </span>
<span class="co">#&gt;  2 PGS000005 PRS313_ERpos </span>
<span class="co">#&gt;  3 PGS000006 PRS313_ERneg </span>
<span class="co">#&gt;  4 PGS000007 PRS3820_BC   </span>
<span class="co">#&gt;  5 PGS000008 PRS3820_ERpos</span>
<span class="co">#&gt;  6 PGS000009 PRS3820_ERneg</span>
<span class="co">#&gt;  7 PGS000001 PRS77_BC     </span>
<span class="co">#&gt;  8 PGS000002 PRS77_ERpos  </span>
<span class="co">#&gt;  9 PGS000003 PRS77_ERneg  </span>
<span class="co">#&gt; 10 PGS000004 PRS313_BC    </span>
<span class="co">#&gt; 11 PGS000005 PRS313_ERpos </span>
<span class="co">#&gt; 12 PGS000006 PRS313_ERneg </span>
<span class="co">#&gt; 13 PGS000007 PRS3820_BC   </span>
<span class="co">#&gt; 14 PGS000008 PRS3820_ERpos</span>
<span class="co">#&gt; 15 PGS000009 PRS3820_ERneg</span></code></pre></div>
<p>From the PGS names we can already see the presence of the suffixes “ERpos” and “ERneg”, suggestive of specialised polygenic risk scores for estrogen-receptor positive and negative samples.</p>
<div class="sourceCode" id="cb11"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">mavaddat2018_scores</span><span class="op">@</span><span class="va">scores</span><span class="op">[</span><span class="st">'scoring_file'</span><span class="op">]</span>
<span class="co">#&gt; # A tibble: 15 x 1</span>
<span class="co">#&gt;    scoring_file                                                                 </span>
<span class="co">#&gt;    &lt;chr&gt;                                                                        </span>
<span class="co">#&gt;  1 http://ftp.ebi.ac.uk/pub/databases/spot/pgs/scores/PGS000004/ScoringFiles/PG…</span>
<span class="co">#&gt;  2 http://ftp.ebi.ac.uk/pub/databases/spot/pgs/scores/PGS000005/ScoringFiles/PG…</span>
<span class="co">#&gt;  3 http://ftp.ebi.ac.uk/pub/databases/spot/pgs/scores/PGS000006/ScoringFiles/PG…</span>
<span class="co">#&gt;  4 http://ftp.ebi.ac.uk/pub/databases/spot/pgs/scores/PGS000007/ScoringFiles/PG…</span>
<span class="co">#&gt;  5 http://ftp.ebi.ac.uk/pub/databases/spot/pgs/scores/PGS000008/ScoringFiles/PG…</span>
<span class="co">#&gt;  6 http://ftp.ebi.ac.uk/pub/databases/spot/pgs/scores/PGS000009/ScoringFiles/PG…</span>
<span class="co">#&gt;  7 http://ftp.ebi.ac.uk/pub/databases/spot/pgs/scores/PGS000001/ScoringFiles/PG…</span>
<span class="co">#&gt;  8 http://ftp.ebi.ac.uk/pub/databases/spot/pgs/scores/PGS000002/ScoringFiles/PG…</span>
<span class="co">#&gt;  9 http://ftp.ebi.ac.uk/pub/databases/spot/pgs/scores/PGS000003/ScoringFiles/PG…</span>
<span class="co">#&gt; 10 http://ftp.ebi.ac.uk/pub/databases/spot/pgs/scores/PGS000004/ScoringFiles/PG…</span>
<span class="co">#&gt; 11 http://ftp.ebi.ac.uk/pub/databases/spot/pgs/scores/PGS000005/ScoringFiles/PG…</span>
<span class="co">#&gt; 12 http://ftp.ebi.ac.uk/pub/databases/spot/pgs/scores/PGS000006/ScoringFiles/PG…</span>
<span class="co">#&gt; 13 http://ftp.ebi.ac.uk/pub/databases/spot/pgs/scores/PGS000007/ScoringFiles/PG…</span>
<span class="co">#&gt; 14 http://ftp.ebi.ac.uk/pub/databases/spot/pgs/scores/PGS000008/ScoringFiles/PG…</span>
<span class="co">#&gt; 15 http://ftp.ebi.ac.uk/pub/databases/spot/pgs/scores/PGS000009/ScoringFiles/PG…</span></code></pre></div>
<p>The column <code>scoring_file</code> contains the URL to the FTP location of the PGS scoring file. PGS scoring files are the files provided by the PGS Catalog that allow you to compute polygenic scores for your samples. You can learn more about scoring files in <code><a href="../articles/pgs-scoring-file.html">vignette("pgs-scoring-file")</a></code>.</p>
<div class="sourceCode" id="cb12"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">mavaddat2018_scores</span><span class="op">@</span><span class="va">scores</span><span class="op">[</span><span class="st">'matches_publication'</span><span class="op">]</span>
<span class="co">#&gt; # A tibble: 15 x 1</span>
<span class="co">#&gt;    matches_publication</span>
<span class="co">#&gt;    &lt;lgl&gt;              </span>
<span class="co">#&gt;  1 TRUE               </span>
<span class="co">#&gt;  2 TRUE               </span>
<span class="co">#&gt;  3 TRUE               </span>
<span class="co">#&gt;  4 TRUE               </span>
<span class="co">#&gt;  5 TRUE               </span>
<span class="co">#&gt;  6 TRUE               </span>
<span class="co">#&gt;  7 TRUE               </span>
<span class="co">#&gt;  8 TRUE               </span>
<span class="co">#&gt;  9 TRUE               </span>
<span class="co">#&gt; 10 TRUE               </span>
<span class="co">#&gt; 11 TRUE               </span>
<span class="co">#&gt; 12 TRUE               </span>
<span class="co">#&gt; 13 TRUE               </span>
<span class="co">#&gt; 14 TRUE               </span>
<span class="co">#&gt; 15 TRUE</span></code></pre></div>
<p>The column <code>matches_publication</code> is a logical value indicating whether the published polygenic score is exactly the same as the one provided to the PGS Catalog. In this case all of the 15 scores are provided exactly as published.</p>
<div class="sourceCode" id="cb13"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">mavaddat2018_scores</span><span class="op">@</span><span class="va">scores</span><span class="op">[</span><span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span><span class="op">(</span><span class="st">'pgs_name'</span>, <span class="st">'pgs_method_name'</span>, <span class="st">'pgs_method_params'</span>, <span class="st">'n_variants'</span>, <span class="st">'assembly'</span><span class="op">)</span><span class="op">]</span>
<span class="co">#&gt; # A tibble: 15 x 5</span>
<span class="co">#&gt;    pgs_name    pgs_method_name              pgs_method_para… n_variants assembly</span>
<span class="co">#&gt;    &lt;chr&gt;       &lt;chr&gt;                        &lt;chr&gt;                 &lt;int&gt; &lt;chr&gt;   </span>
<span class="co">#&gt;  1 PRS313_BC   Hard-Thresholding Stepwise … p &lt; 10^-5               313 GRCh37  </span>
<span class="co">#&gt;  2 PRS313_ERp… Hard-Thresholding Stepwise … p &lt; 10^-5               313 GRCh37  </span>
<span class="co">#&gt;  3 PRS313_ERn… Hard-Thresholding Stepwise … p &lt; 10^-5               313 GRCh37  </span>
<span class="co">#&gt;  4 PRS3820_BC  LASSO penalized regression   p &lt; 0.001              3820 GRCh37  </span>
<span class="co">#&gt;  5 PRS3820_ER… LASSO penalized regression   p &lt; 0.001              3820 GRCh37  </span>
<span class="co">#&gt;  6 PRS3820_ER… LASSO penalized regression   p &lt; 0.001              3820 GRCh37  </span>
<span class="co">#&gt;  7 PRS77_BC    SNPs passing genome-wide si… P&lt;5x10-8                 77 NR      </span>
<span class="co">#&gt;  8 PRS77_ERpos SNPs passing genome-wide si… P&lt;5x10-8                 77 NR      </span>
<span class="co">#&gt;  9 PRS77_ERneg SNPs passing genome-wide si… P&lt;5x10-8                 77 NR      </span>
<span class="co">#&gt; 10 PRS313_BC   Hard-Thresholding Stepwise … p &lt; 10^-5               313 GRCh37  </span>
<span class="co">#&gt; 11 PRS313_ERp… Hard-Thresholding Stepwise … p &lt; 10^-5               313 GRCh37  </span>
<span class="co">#&gt; 12 PRS313_ERn… Hard-Thresholding Stepwise … p &lt; 10^-5               313 GRCh37  </span>
<span class="co">#&gt; 13 PRS3820_BC  LASSO penalized regression   p &lt; 0.001              3820 GRCh37  </span>
<span class="co">#&gt; 14 PRS3820_ER… LASSO penalized regression   p &lt; 0.001              3820 GRCh37  </span>
<span class="co">#&gt; 15 PRS3820_ER… LASSO penalized regression   p &lt; 0.001              3820 GRCh37</span></code></pre></div>
<p>In addition, columns such as <code>pgs_method_name</code> and <code>pgs_method_params</code> provide extra details about the PGS development method. Finally, <code>n_variants</code> informs about the number of variants comprising each polygenic risk score, and <code>assembly</code> indicates the genome assembly.</p>
</div>
<div id="publications-table" class="section level3">
<h3 class="hasAnchor">
<a href="#publications-table" class="anchor"></a>Publications table</h3>
<p>The S4 scores object contains a table about the publications. In this case, it is not surprising that all PGS scores map to the same publication identifier, i.e., PGP000002, PGP000001, as that was our starting point.</p>
<div class="sourceCode" id="cb14"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">mavaddat2018_scores</span><span class="op">@</span><span class="va">publications</span><span class="op">[</span><span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span><span class="op">(</span><span class="st">'pgs_id'</span>, <span class="st">'pgp_id'</span>, <span class="st">'publication_date'</span>, <span class="st">'author_fullname'</span><span class="op">)</span><span class="op">]</span>
<span class="co">#&gt; # A tibble: 15 x 4</span>
<span class="co">#&gt;    pgs_id    pgp_id    publication_date author_fullname</span>
<span class="co">#&gt;    &lt;chr&gt;     &lt;chr&gt;     &lt;date&gt;           &lt;chr&gt;          </span>
<span class="co">#&gt;  1 PGS000004 PGP000002 2018-12-13       Mavaddat N     </span>
<span class="co">#&gt;  2 PGS000005 PGP000002 2018-12-13       Mavaddat N     </span>
<span class="co">#&gt;  3 PGS000006 PGP000002 2018-12-13       Mavaddat N     </span>
<span class="co">#&gt;  4 PGS000007 PGP000002 2018-12-13       Mavaddat N     </span>
<span class="co">#&gt;  5 PGS000008 PGP000002 2018-12-13       Mavaddat N     </span>
<span class="co">#&gt;  6 PGS000009 PGP000002 2018-12-13       Mavaddat N     </span>
<span class="co">#&gt;  7 PGS000001 PGP000001 2015-04-08       Mavaddat N     </span>
<span class="co">#&gt;  8 PGS000002 PGP000001 2015-04-08       Mavaddat N     </span>
<span class="co">#&gt;  9 PGS000003 PGP000001 2015-04-08       Mavaddat N     </span>
<span class="co">#&gt; 10 PGS000004 PGP000002 2018-12-13       Mavaddat N     </span>
<span class="co">#&gt; 11 PGS000005 PGP000002 2018-12-13       Mavaddat N     </span>
<span class="co">#&gt; 12 PGS000006 PGP000002 2018-12-13       Mavaddat N     </span>
<span class="co">#&gt; 13 PGS000007 PGP000002 2018-12-13       Mavaddat N     </span>
<span class="co">#&gt; 14 PGS000008 PGP000002 2018-12-13       Mavaddat N     </span>
<span class="co">#&gt; 15 PGS000009 PGP000002 2018-12-13       Mavaddat N</span></code></pre></div>
</div>
<div id="samples-table" class="section level3">
<h3 class="hasAnchor">
<a href="#samples-table" class="anchor"></a>Samples table</h3>
<p>The third table (slot) of the S4 scores object pertains the samples used in the development of the PGS scores.</p>
<div class="sourceCode" id="cb15"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">mavaddat2018_scores</span><span class="op">@</span><span class="va">samples</span>
<span class="co">#&gt; # A tibble: 39 x 15</span>
<span class="co">#&gt;    pgs_id sample_id stage sample_size sample_cases sample_controls</span>
<span class="co">#&gt;    &lt;chr&gt;      &lt;int&gt; &lt;chr&gt;       &lt;int&gt;        &lt;int&gt;           &lt;int&gt;</span>
<span class="co">#&gt;  1 PGS00…         1 disc…      139274           NA              NA</span>
<span class="co">#&gt;  2 PGS00…         2 trai…      158648        88916           69732</span>
<span class="co">#&gt;  3 PGS00…         3 trai…       10444         5159            5285</span>
<span class="co">#&gt;  4 PGS00…         1 disc…      139274           NA              NA</span>
<span class="co">#&gt;  5 PGS00…         2 trai…       87368        55391           31977</span>
<span class="co">#&gt;  6 PGS00…         3 trai…        5159         4233             926</span>
<span class="co">#&gt;  7 PGS00…         1 disc…      139274           NA              NA</span>
<span class="co">#&gt;  8 PGS00…         2 trai…       87368        15404           71964</span>
<span class="co">#&gt;  9 PGS00…         3 trai…        5159          926            4233</span>
<span class="co">#&gt; 10 PGS00…         1 disc…      139274           NA              NA</span>
<span class="co">#&gt; # … with 29 more rows, and 9 more variables: sample_percent_male &lt;dbl&gt;,</span>
<span class="co">#&gt; #   phenotype_description &lt;chr&gt;, ancestry &lt;chr&gt;, ancestry_description &lt;chr&gt;,</span>
<span class="co">#&gt; #   ancestry_country &lt;chr&gt;, ancestry_additional_description &lt;chr&gt;,</span>
<span class="co">#&gt; #   study_id &lt;chr&gt;, pubmed_id &lt;chr&gt;, cohorts_additional_description &lt;chr&gt;</span></code></pre></div>
<p>TODO: waiting feedback from the PGS Catalog team on sample “stage”. TODO: Add here link to a vignette about samples.</p>
</div>
<div id="demographics-table" class="section level3">
<h3 class="hasAnchor">
<a href="#demographics-table" class="anchor"></a>Demographics table</h3>
<p>The demographics table usually lists demographic information about each sample. For this study this table is however empty, meaning that this information was not available from Mavaddat’s.</p>
<p>TODO: Link here another vignette with an example for which all slots are not empty.</p>
<div class="sourceCode" id="cb16"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">mavaddat2018_scores</span><span class="op">@</span><span class="va">demographics</span>
<span class="co">#&gt; # A tibble: 0 x 11</span>
<span class="co">#&gt; # … with 11 variables: pgs_id &lt;chr&gt;, sample_id &lt;int&gt;, variable &lt;chr&gt;,</span>
<span class="co">#&gt; #   estimate_type &lt;chr&gt;, estimate &lt;dbl&gt;, unit &lt;chr&gt;, variability_type &lt;chr&gt;,</span>
<span class="co">#&gt; #   variability &lt;dbl&gt;, interval_type &lt;chr&gt;, interval_lower &lt;dbl&gt;,</span>
<span class="co">#&gt; #   interval_upper &lt;dbl&gt;</span></code></pre></div>
</div>
<div id="cohorts-table" class="section level3">
<h3 class="hasAnchor">
<a href="#cohorts-table" class="anchor"></a>Cohorts table</h3>
<p>In the cohorts table you can check which cohorts belong to which samples. Note that the unique identification of a sample is given by the combination of the values of the first two columns: <code>pgs_id</code>, and <code>sample_id</code>.</p>
<p>TODO: Add here link to a vignette about samples.</p>
<div class="sourceCode" id="cb17"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">mavaddat2018_scores</span><span class="op">@</span><span class="va">cohorts</span>
<span class="co">#&gt; # A tibble: 1,116 x 4</span>
<span class="co">#&gt;    pgs_id    sample_id cohort_symbol cohort_name                                </span>
<span class="co">#&gt;    &lt;chr&gt;         &lt;int&gt; &lt;chr&gt;         &lt;chr&gt;                                      </span>
<span class="co">#&gt;  1 PGS000004         2 ABCFS         Australian Breast Cancer Family Study      </span>
<span class="co">#&gt;  2 PGS000004         2 MCCS          Melbourne Collaborative Cohort Study       </span>
<span class="co">#&gt;  3 PGS000004         2 HMBCS         Hannover-Minsk Breast Cancer Study         </span>
<span class="co">#&gt;  4 PGS000004         2 LMBC          Leuven Multidisciplinary Breast Centre     </span>
<span class="co">#&gt;  5 PGS000004         2 MTLGEBCS      Montreal Gene-Environment Breast Cancer St…</span>
<span class="co">#&gt;  6 PGS000004         2 CGPS          Copenhagen General Population Study        </span>
<span class="co">#&gt;  7 PGS000004         2 KBCP          Kuopio Breast Cancer Project               </span>
<span class="co">#&gt;  8 PGS000004         2 OBCS          Oulu Breast Cancer Study                   </span>
<span class="co">#&gt;  9 PGS000004         2 CECILE        CECILE Breast Cancer Study                 </span>
<span class="co">#&gt; 10 PGS000004         2 BBCC          Bavarian Breast Cancer Cases and Controls  </span>
<span class="co">#&gt; # … with 1,106 more rows</span></code></pre></div>
</div>
<div id="traits-table" class="section level3">
<h3 class="hasAnchor">
<a href="#traits-table" class="anchor"></a>Traits table</h3>
<p>Finally, in the traits table, you have access to the associated traits with these polygenic scores. With no surprise, all scores indicate breast cancer or a subtype thereof (column <code>trait</code>):</p>
<div class="sourceCode" id="cb18"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">mavaddat2018_scores</span><span class="op">@</span><span class="va">traits</span><span class="op">[</span><span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span><span class="op">(</span><span class="st">'pgs_id'</span>, <span class="st">'efo_id'</span>, <span class="st">'trait'</span><span class="op">)</span><span class="op">]</span>
<span class="co">#&gt; # A tibble: 15 x 3</span>
<span class="co">#&gt;    pgs_id    efo_id      trait                                   </span>
<span class="co">#&gt;    &lt;chr&gt;     &lt;chr&gt;       &lt;chr&gt;                                   </span>
<span class="co">#&gt;  1 PGS000004 EFO_0000305 breast carcinoma                        </span>
<span class="co">#&gt;  2 PGS000005 EFO_1000649 estrogen-receptor positive breast cancer</span>
<span class="co">#&gt;  3 PGS000006 EFO_1000650 estrogen-receptor negative breast cancer</span>
<span class="co">#&gt;  4 PGS000007 EFO_0000305 breast carcinoma                        </span>
<span class="co">#&gt;  5 PGS000008 EFO_1000649 estrogen-receptor positive breast cancer</span>
<span class="co">#&gt;  6 PGS000009 EFO_1000650 estrogen-receptor negative breast cancer</span>
<span class="co">#&gt;  7 PGS000001 EFO_0000305 breast carcinoma                        </span>
<span class="co">#&gt;  8 PGS000002 EFO_1000649 estrogen-receptor positive breast cancer</span>
<span class="co">#&gt;  9 PGS000003 EFO_1000650 estrogen-receptor negative breast cancer</span>
<span class="co">#&gt; 10 PGS000004 EFO_0000305 breast carcinoma                        </span>
<span class="co">#&gt; 11 PGS000005 EFO_1000649 estrogen-receptor positive breast cancer</span>
<span class="co">#&gt; 12 PGS000006 EFO_1000650 estrogen-receptor negative breast cancer</span>
<span class="co">#&gt; 13 PGS000007 EFO_0000305 breast carcinoma                        </span>
<span class="co">#&gt; 14 PGS000008 EFO_1000649 estrogen-receptor positive breast cancer</span>
<span class="co">#&gt; 15 PGS000009 EFO_1000650 estrogen-receptor negative breast cancer</span></code></pre></div>
<p>Compared to the author-reported trait (column <code>reported_trait</code> from table <code>scores</code>), the trait description in this table follows the controlled vocabulary of an ontology, i.e., the <a href="https://www.ebi.ac.uk/efo/">Experimental Factor Ontology (EFO)</a>. This way, traits are described objectively. This is very useful for comparing trait data among different studies where different reported trait descriptions might have been used. For example, if you wanted now to know what other polygenic scores may be deposited in the PGS Catalog that also study breast cancer — namely, breast carcinoma, estrogen-receptor positive breast cancer, and estrogen-receptor negative breast cancer — then you could use their respective EFO identifiers (EFO_0000305, EFO_1000649, and EFO_1000650) with the function <code><a href="../reference/get_scores.html">get_scores()</a></code>:</p>
<div class="sourceCode" id="cb19"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">scores_bc</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/get_scores.html">get_scores</a></span><span class="op">(</span>efo_id <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/unique.html">unique</a></span><span class="op">(</span><span class="va">mavaddat2018_scores</span><span class="op">@</span><span class="va">traits</span><span class="op">[[</span><span class="st">'efo_id'</span><span class="op">]</span><span class="op">]</span><span class="op">)</span><span class="op">)</span>
<span class="fu"><a href="../reference/n.html">n</a></span><span class="op">(</span><span class="va">scores_bc</span><span class="op">)</span>
<span class="co">#&gt; [1] 96</span></code></pre></div>
<p>So there are 96 scores present in the Catalog. Included in this set are the 15 scores originating from <a href="https://doi.org/10.1016/j.ajhg.2018.11.002">Mavaddat et al. (2018)</a>. If we wanted to proceed in analysing these other scores without those of Mavaddat’s study, we could use the function <code><a href="../reference/setop.html">setdiff()</a></code> to remove them from the S4 scores object:</p>
<div class="sourceCode" id="cb20"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">bc_scores_not_mavaddat2018</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/setop.html">setdiff</a></span><span class="op">(</span><span class="va">scores_bc</span>, <span class="va">mavaddat2018_scores</span><span class="op">)</span>
<span class="fu"><a href="../reference/n.html">n</a></span><span class="op">(</span><span class="va">bc_scores_not_mavaddat2018</span><span class="op">)</span>
<span class="co">#&gt; [1] 87</span></code></pre></div>
<p>For other set operations check their documentation page: <code><a href="../reference/setop.html">union()</a></code>, <code><a href="../reference/setop.html">intersect()</a></code> and <code><a href="../reference/setop.html">setequal()</a></code>.</p>
</div>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">

        <nav id="toc" data-toggle="toc"><h2 data-toc-skip>Contents</h2>
    </nav>
</div>

</div>



      <footer><div class="copyright">
  <p>Developed by <a href="https://rmagno.eu/">Ramiro Magno</a>, <a href="https://iduarte.eu/">Isabel Duarte</a>, Ana-Teresa Maia, Centre for Biomedical Research, University of Algarve.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.6.1.</p>
</div>

      </footer>
</div>

  


  </body>
</html>
