<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<title>PGS scoring file • quincunx</title>
<!-- favicons --><link rel="icon" type="image/png" sizes="96x96" href="../favicon-96x96.png">
<link rel="icon" type="”image/svg+xml”" href="../favicon.svg">
<link rel="apple-touch-icon" sizes="180x180" href="../apple-touch-icon.png">
<link rel="icon" sizes="any" href="../favicon.ico">
<link rel="manifest" href="../site.webmanifest">
<script src="../deps/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="../deps/bootstrap-5.3.1/bootstrap.min.css" rel="stylesheet">
<script src="../deps/bootstrap-5.3.1/bootstrap.bundle.min.js"></script><link href="../deps/Nunito-0.4.10/font.css" rel="stylesheet">
<link href="../deps/Inconsolata-0.4.10/font.css" rel="stylesheet">
<link href="../deps/Source_Sans_Pro-0.4.10/font.css" rel="stylesheet">
<link href="../deps/font-awesome-6.5.2/css/all.min.css" rel="stylesheet">
<link href="../deps/font-awesome-6.5.2/css/v4-shims.min.css" rel="stylesheet">
<script src="../deps/headroom-0.11.0/headroom.min.js"></script><script src="../deps/headroom-0.11.0/jQuery.headroom.min.js"></script><script src="../deps/bootstrap-toc-1.0.1/bootstrap-toc.min.js"></script><script src="../deps/clipboard.js-2.0.11/clipboard.min.js"></script><script src="../deps/search-1.0.0/autocomplete.jquery.min.js"></script><script src="../deps/search-1.0.0/fuse.min.js"></script><script src="../deps/search-1.0.0/mark.min.js"></script><!-- pkgdown --><script src="../pkgdown.js"></script><meta property="og:title" content="PGS scoring file">
</head>
<body>
    <a href="#main" class="visually-hidden-focusable">Skip to contents</a>


    <nav class="navbar navbar-expand-lg fixed-top bg-none" data-bs-theme="light" aria-label="Site navigation"><div class="container">

    <a class="navbar-brand me-2" href="../index.html">quincunx</a>

    <small class="nav-text text-muted me-auto" data-bs-toggle="tooltip" data-bs-placement="bottom" title="">0.1.10</small>


    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div id="navbar" class="collapse navbar-collapse ms-3">
      <ul class="navbar-nav me-auto">
<li class="nav-item"><a class="nav-link" href="../reference/index.html">Reference</a></li>
<li class="active nav-item dropdown">
  <button class="nav-link dropdown-toggle" type="button" id="dropdown-articles" data-bs-toggle="dropdown" aria-expanded="false" aria-haspopup="true">Articles</button>
  <ul class="dropdown-menu" aria-labelledby="dropdown-articles">
<li><hr class="dropdown-divider"></li>
    <li><h6 class="dropdown-header" data-toc-skip>PGS Catalog Concepts</h6></li>
    <li><a class="dropdown-item" href="../articles/identifiers.html">Identifiers</a></li>
    <li><a class="dropdown-item" href="../articles/pgs-construction-process.html">PGS Construction Process and Stages</a></li>
    <li><a class="dropdown-item" href="../articles/cohorts-samples-sample-sets.html">Cohorts, Samples and Sample Sets</a></li>
    <li><hr class="dropdown-divider"></li>
    <li><h6 class="dropdown-header" data-toc-skip>Getting PGS Catalog Data</h6></li>
    <li><a class="dropdown-item" href="../articles/getting-pgs-scores.html">Getting PGS Scores</a></li>
    <li><a class="dropdown-item" href="../articles/getting-pgs-publications.html">Getting PGS Publications</a></li>
    <li><a class="dropdown-item" href="../articles/getting-pgs-sample-sets.html">Getting PGS Sample Sets</a></li>
    <li><a class="dropdown-item" href="../articles/getting-pgs-performance-metrics.html">Getting PGS Performance Metrics</a></li>
    <li><a class="dropdown-item" href="../articles/getting-traits.html">Getting Traits</a></li>
    <li><a class="dropdown-item" href="../articles/getting-cohorts.html">Getting Cohorts</a></li>
    <li><a class="dropdown-item" href="../articles/getting-releases.html">PGS Catalog Releases</a></li>
    <li><a class="dropdown-item" href="../articles/pgs-scoring-file.html">PGS scoring file</a></li>
    <li><hr class="dropdown-divider"></li>
    <li><h6 class="dropdown-header" data-toc-skip>Wrangling PGS Catalog Data</h6></li>
    <li><a class="dropdown-item" href="../articles/pgs-cat-ent-subsetting.html">PGS Catalog Entity Subsetting</a></li>
    <li><hr class="dropdown-divider"></li>
    <li><h6 class="dropdown-header" data-toc-skip>Case Studies</h6></li>
    <li><a class="dropdown-item" href="../articles/risk-strat-brs-patients.html">Risk of Brugada syndrome patients</a></li>
    <li><a class="dropdown-item" href="../articles/pgs-scores-mavaddat.html">Exploring PGS scores by Mavaddat et al. (2018)</a></li>
    <li><hr class="dropdown-divider"></li>
    <li><a class="dropdown-item" href="../articles/index.html">More articles...</a></li>
  </ul>
</li>
<li class="nav-item"><a class="nav-link" href="../news/index.html">Changelog</a></li>
      </ul>
<ul class="navbar-nav">
<li class="nav-item"><form class="form-inline" role="search">
 <input class="form-control" type="search" name="search-input" id="search-input" autocomplete="off" aria-label="Search site" placeholder="Search for" data-search-index="../search.json">
</form></li>
<li class="nav-item"><a class="external-link nav-link" href="https://github.com/ramiromagno/quincunx/" aria-label="GitHub"><span class="fa fab fa-github fa-lg"></span></a></li>
      </ul>
</div>


  </div>
</nav><div class="container template-article">




<div class="row">
  <main id="main" class="col-md-9"><div class="page-header">
      <img src="../logo.svg" class="logo" alt=""><h1>PGS scoring file</h1>
            
      
      <small class="dont-index">Source: <a href="https://github.com/ramiromagno/quincunx/blob/master/vignettes/pgs-scoring-file.Rmd" class="external-link"><code>vignettes/pgs-scoring-file.Rmd</code></a></small>
      <div class="d-none name"><code>pgs-scoring-file.Rmd</code></div>
    </div>

    
    
<div class="section level2">
<h2 id="introduction">Introduction<a class="anchor" aria-label="anchor" href="#introduction"></a>
</h2>
<p>PGS scoring files are the files provided by the PGS Catalog that
allow you to compute polygenic scores for your samples. These files are
hosted at the PGS Catalog FTP server: <a href="http://ftp.ebi.ac.uk/pub/databases/spot/pgs/scores/" class="external-link">http://ftp.ebi.ac.uk/pub/databases/spot/pgs/scores/</a>.
They are labelled by their respective PGS Score ID
(e.g. <code>PGS000001.txt.gz</code>).</p>
<p>Essentially, each scoring file contains variant identification,
effect alleles and respective score weights. The file is formatted to be
a gzipped tab-delimited text file. The file starts with an header that
contains brief metadata about the score.</p>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1" tabindex="-1"></a><span class="do">###PGS CATALOG SCORING FILE - see https://www.pgscatalog.org/downloads/#dl_ftp_scoring for additional information</span></span>
<span id="cb1-2"><a href="#cb1-2" tabindex="-1"></a><span class="co">#format_version=2.0</span></span>
<span id="cb1-3"><a href="#cb1-3" tabindex="-1"></a><span class="do">##POLYGENIC SCORE (PGS) INFORMATION</span></span>
<span id="cb1-4"><a href="#cb1-4" tabindex="-1"></a><span class="co">#pgs_id=PGS000117</span></span>
<span id="cb1-5"><a href="#cb1-5" tabindex="-1"></a><span class="co">#pgs_name=CVD_EJ2020</span></span>
<span id="cb1-6"><a href="#cb1-6" tabindex="-1"></a><span class="co">#trait_reported=Cardiovascular Disease </span></span>
<span id="cb1-7"><a href="#cb1-7" tabindex="-1"></a><span class="co">#trait_mapped=cardiovascular disease</span></span>
<span id="cb1-8"><a href="#cb1-8" tabindex="-1"></a><span class="co">#trait_efo=EFO_0000319</span></span>
<span id="cb1-9"><a href="#cb1-9" tabindex="-1"></a><span class="co">#weight_type=NR</span></span>
<span id="cb1-10"><a href="#cb1-10" tabindex="-1"></a><span class="co">#genome_build=GRCh37</span></span>
<span id="cb1-11"><a href="#cb1-11" tabindex="-1"></a><span class="co">#variants_number=267863</span></span>
<span id="cb1-12"><a href="#cb1-12" tabindex="-1"></a><span class="do">##SOURCE INFORMATION</span></span>
<span id="cb1-13"><a href="#cb1-13" tabindex="-1"></a><span class="co">#pgp_id=PGP000054</span></span>
<span id="cb1-14"><a href="#cb1-14" tabindex="-1"></a><span class="co">#citation=Elliott J et al. JAMA (2020). doi:10.1001/jama.2019.22241</span></span>
<span id="cb1-15"><a href="#cb1-15" tabindex="-1"></a>rsID    chr_name    chr_position    effect_allele   other_allele    effect_weight</span>
<span id="cb1-16"><a href="#cb1-16" tabindex="-1"></a>rs11240779  <span class="dv">1</span>   <span class="dv">808631</span>  A   G   <span class="fl">0.00077622</span></span>
<span id="cb1-17"><a href="#cb1-17" tabindex="-1"></a>rs1921  <span class="dv">1</span>   <span class="dv">949608</span>  A   G   <span class="sc">-</span><span class="fl">0.00583829</span></span>
<span id="cb1-18"><a href="#cb1-18" tabindex="-1"></a>rs2710890   <span class="dv">1</span>   <span class="dv">958905</span>  G   A   <span class="sc">-</span><span class="fl">0.00182583</span></span>
<span id="cb1-19"><a href="#cb1-19" tabindex="-1"></a>rs4970349   <span class="dv">1</span>   <span class="dv">967658</span>  T   C   <span class="sc">-</span><span class="fl">0.001855691</span></span>
<span id="cb1-20"><a href="#cb1-20" tabindex="-1"></a>rs2710887   <span class="dv">1</span>   <span class="dv">986443</span>  T   C   <span class="sc">-</span><span class="fl">0.003430246</span></span></code></pre></div>
</div>
<div class="section level2">
<h2 id="getting-pgs-scoring-files">Getting PGS scoring files<a class="anchor" aria-label="anchor" href="#getting-pgs-scoring-files"></a>
</h2>
<p>To get PGS scoring files you may either download them directly from
the FTP server yourself or use the quincunx function
<code><a href="../reference/read_scoring_file.html">read_scoring_file()</a></code>. This function automatically downloads,
unzips and reads both the header and the data as a data frame into R.
Let’s look at an example: polygenic score “PGS000655”. You can use
directly the PGS score ID as the argument to the <code>source</code>
parameter in the function <code><a href="../reference/read_scoring_file.html">read_scoring_file()</a></code>, and it will
translate it to the corresponding URL, download the file and import the
data as data frame with appropriate data types.</p>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">my_pgs</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/read_scoring_file.html">read_scoring_file</a></span><span class="op">(</span>source <span class="op">=</span> <span class="st">"PGS000655"</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/length.html" class="external-link">length</a></span><span class="op">(</span><span class="va">my_pgs</span><span class="op">)</span></span>
<span><span class="co">#&gt; [1] 1</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/names.html" class="external-link">names</a></span><span class="op">(</span><span class="va">my_pgs</span><span class="op">)</span></span>
<span><span class="co">#&gt; [1] "PGS000655"</span></span></code></pre></div>
<p>The returned object of <code><a href="../reference/read_scoring_file.html">read_scoring_file()</a></code> is a list of
lists. The outer list runs (potentially) over several PGS IDs if you
asked for more than one. So we start by extracting the first (and in
this case only) score by name:</p>
<div class="sourceCode" id="cb3"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">PGS000655</span> <span class="op">&lt;-</span> <span class="va">my_pgs</span><span class="op">[[</span><span class="st">'PGS000655'</span><span class="op">]</span><span class="op">]</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/names.html" class="external-link">names</a></span><span class="op">(</span><span class="va">PGS000655</span><span class="op">)</span></span>
<span><span class="co">#&gt; [1] "metadata" "data"</span></span></code></pre></div>
<p>Now, each inner list comprises two elements: <code>metadata</code>
and <code>data</code>. The metadata part contains the information
retrieved from the initial comment block of the file:</p>
<div class="sourceCode" id="cb4"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://pillar.r-lib.org/reference/glimpse.html" class="external-link">glimpse</a></span><span class="op">(</span><span class="va">PGS000655</span><span class="op">$</span><span class="va">metadata</span><span class="op">)</span></span>
<span><span class="co">#&gt; Rows: 1</span></span>
<span><span class="co">#&gt; Columns: 16</span></span>
<span><span class="co">#&gt; $ format_version     <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> "2.0"</span></span>
<span><span class="co">#&gt; $ pgs_id             <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> "PGS000655"</span></span>
<span><span class="co">#&gt; $ pgs_name           <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> "NAFLD-10"</span></span>
<span><span class="co">#&gt; $ reported_trait     <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> "Non-alcoholic fatty liver disease"</span></span>
<span><span class="co">#&gt; $ mapped_trait       <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> "non-alcoholic fatty liver disease"</span></span>
<span><span class="co">#&gt; $ efo_trait          <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> "EFO_0003095"</span></span>
<span><span class="co">#&gt; $ genome_build       <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> "GRCh37"</span></span>
<span><span class="co">#&gt; $ weight_type        <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> "beta"</span></span>
<span><span class="co">#&gt; $ number_of_variants <span style="color: #949494; font-style: italic;">&lt;int&gt;</span> 10</span></span>
<span><span class="co">#&gt; $ pgp_id             <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> "PGP000119"</span></span>
<span><span class="co">#&gt; $ citation           <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> "Namjou B et al. BMC Med (2019). doi:10.1186/s12916…</span></span>
<span><span class="co">#&gt; $ license            <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> <span style="color: #BB0000;">NA</span></span></span>
<span><span class="co">#&gt; $ HmPOS_build        <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> <span style="color: #BB0000;">NA</span></span></span>
<span><span class="co">#&gt; $ HmPOS_date         <span style="color: #949494; font-style: italic;">&lt;date&gt;</span> <span style="color: #BB0000;">NA</span></span></span>
<span><span class="co">#&gt; $ HmPOS_match_chr    <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> <span style="color: #BB0000;">NA</span></span></span>
<span><span class="co">#&gt; $ HmPOS_match_pos    <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> <span style="color: #BB0000;">NA</span></span></span></code></pre></div>
<p>From the metadata we can see that this score is for the trait
<em>Non-alcoholic fatty liver disease</em>, and that it is composed of
10 variants. Remember that you can retrieve more metadata information
about this score by using <code>get_scores('PGS000655')</code> or
<code>get_performance_metrics('PGS000655')</code>.</p>
<p>The data element contains the actual relevant information for
calculating scores:</p>
<div class="sourceCode" id="cb5"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">PGS000655</span><span class="op">$</span><span class="va">data</span></span>
<span><span class="co">#&gt; <span style="color: #949494;"># A tibble: 10 × 4</span></span></span>
<span><span class="co">#&gt;    rsID       effect_allele effect_weight    OR</span></span>
<span><span class="co">#&gt;    <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>      <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>                 <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span> <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span></span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 1</span> rs1260326  T                    0.057<span style="text-decoration: underline;">1</span>  1.14</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 2</span> rs780094   T                    0.061<span style="text-decoration: underline;">1</span>  1.15</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 3</span> rs2954021  G                   -<span style="color: #BB0000;">0.098</span><span style="color: #BB0000; text-decoration: underline;">0</span>  0.8 </span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 4</span> rs4808199  A                    0.086<span style="text-decoration: underline;">8</span>  1.22</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 5</span> rs58542926 T                    0.089<span style="text-decoration: underline;">9</span>  1.23</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 6</span> rs738409   G                    0.253   1.79</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 7</span> rs3747207  A                    0.250   1.78</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 8</span> rs2294915  T                    0.243   1.75</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 9</span> rs1227756  G                    0.026<span style="text-decoration: underline;">1</span>  1.06</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">10</span> rs2294918  A                   -<span style="color: #BB0000;">0.114</span>   0.78</span></span></code></pre></div>
<p>The minimal information for calculating scores are: variant
identification information, and its effect allele, as well as the
corresponding score weight. Currently, the PGS Catalog project,
identifies a variant by either its rsID (column <code>rsID</code>), or
by its genomic position (columns <code>chr_name</code> and
<code>chr_position</code>). The effect allele and the score weight are
given in the columns <code>effect_allele</code> and
<code>effect_weight</code>, respectively.</p>
<div class="sourceCode" id="cb6"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">PGS000655</span><span class="op">$</span><span class="va">data</span><span class="op">[</span><span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">'rsID'</span>, <span class="st">'effect_allele'</span>, <span class="st">'effect_weight'</span><span class="op">)</span><span class="op">]</span></span>
<span><span class="co">#&gt; <span style="color: #949494;"># A tibble: 10 × 3</span></span></span>
<span><span class="co">#&gt;    rsID       effect_allele effect_weight</span></span>
<span><span class="co">#&gt;    <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>      <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>                 <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span></span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 1</span> rs1260326  T                    0.057<span style="text-decoration: underline;">1</span></span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 2</span> rs780094   T                    0.061<span style="text-decoration: underline;">1</span></span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 3</span> rs2954021  G                   -<span style="color: #BB0000;">0.098</span><span style="color: #BB0000; text-decoration: underline;">0</span></span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 4</span> rs4808199  A                    0.086<span style="text-decoration: underline;">8</span></span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 5</span> rs58542926 T                    0.089<span style="text-decoration: underline;">9</span></span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 6</span> rs738409   G                    0.253 </span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 7</span> rs3747207  A                    0.250 </span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 8</span> rs2294915  T                    0.243 </span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 9</span> rs1227756  G                    0.026<span style="text-decoration: underline;">1</span></span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">10</span> rs2294918  A                   -<span style="color: #BB0000;">0.114</span></span></span></code></pre></div>
</div>
<div class="section level2">
<h2 id="note-about-large-pgs-files">Note about large PGS files<a class="anchor" aria-label="anchor" href="#note-about-large-pgs-files"></a>
</h2>
<p>Please note that number of variants comprising PGS scores can vary
wildly. In the example above (<code>"PGS000655"</code>) we provided a
very small example. Other scores may be composed of millions of
variants, e.g., <code>"PGS000013"</code>. Reading this scoring files
directly to memory may take up a substantial amount of resources (in the
order of Gigabytes). So to prevent stalling your computer while reading
these files we recommend to read in only the metadata, as this can be
done efficiently. As the number of variants is indicated in the metadata
then you can decide whether to read in all the information or not.</p>
<p>To read only the metadata set the parameter
<code>metadata_only</code> to <code>TRUE</code>:</p>
<div class="sourceCode" id="cb7"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">my_big_scoring_file</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/read_scoring_file.html">read_scoring_file</a></span><span class="op">(</span>source <span class="op">=</span> <span class="st">"PGS000013"</span>, metadata_only <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span></span>
<span><span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://pillar.r-lib.org/reference/glimpse.html" class="external-link">glimpse</a></span><span class="op">(</span><span class="va">my_big_scoring_file</span><span class="op">[[</span><span class="st">'PGS000013'</span><span class="op">]</span><span class="op">]</span><span class="op">$</span><span class="va">metadata</span><span class="op">)</span></span>
<span><span class="co">#&gt; Rows: 1</span></span>
<span><span class="co">#&gt; Columns: 16</span></span>
<span><span class="co">#&gt; $ format_version     <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> "2.0"</span></span>
<span><span class="co">#&gt; $ pgs_id             <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> "PGS000013"</span></span>
<span><span class="co">#&gt; $ pgs_name           <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> "GPS_CAD"</span></span>
<span><span class="co">#&gt; $ reported_trait     <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> "Coronary artery disease"</span></span>
<span><span class="co">#&gt; $ mapped_trait       <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> "coronary artery disease"</span></span>
<span><span class="co">#&gt; $ efo_trait          <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> "EFO_0001645"</span></span>
<span><span class="co">#&gt; $ genome_build       <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> "hg19"</span></span>
<span><span class="co">#&gt; $ weight_type        <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> <span style="color: #BB0000;">NA</span></span></span>
<span><span class="co">#&gt; $ number_of_variants <span style="color: #949494; font-style: italic;">&lt;int&gt;</span> 6630150</span></span>
<span><span class="co">#&gt; $ pgp_id             <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> "PGP000006"</span></span>
<span><span class="co">#&gt; $ citation           <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> "Khera AV et al. Nat Genet (2018). doi:10.1038/s415…</span></span>
<span><span class="co">#&gt; $ license            <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> "Freely available to the academic community for res…</span></span>
<span><span class="co">#&gt; $ HmPOS_build        <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> <span style="color: #BB0000;">NA</span></span></span>
<span><span class="co">#&gt; $ HmPOS_date         <span style="color: #949494; font-style: italic;">&lt;date&gt;</span> <span style="color: #BB0000;">NA</span></span></span>
<span><span class="co">#&gt; $ HmPOS_match_chr    <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> <span style="color: #BB0000;">NA</span></span></span>
<span><span class="co">#&gt; $ HmPOS_match_pos    <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> <span style="color: #BB0000;">NA</span></span></span></code></pre></div>
<p>As can be seen from the variable <code>number_of_variants</code>, the
number of variants is greater than 6 million. This means that reading
this PGS scoring file has these many rows which can make it burdensome
on some computers.</p>
<p>To know how many variants are included in every score, you can take
advantage of the <code><a href="../reference/get_scores.html">get_scores()</a></code> function:</p>
<div class="sourceCode" id="cb8"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co"># all_scores &lt;- get_scores()</span></span>
<span><span class="co"># all_scores@scores[c('pgs_id', 'n_variants')]</span></span></code></pre></div>
</div>
<div class="section level2">
<h2 id="pgs-scoring-file-format">PGS scoring file format<a class="anchor" aria-label="anchor" href="#pgs-scoring-file-format"></a>
</h2>
<p>Each scoring file contains variant identification, effect alleles and
respective score weights. The file is formatted as a gzipped
tab-delimited text file, with a header containing brief metadata about
the score.</p>
<div class="section level3">
<h3 id="header">Header<a class="anchor" aria-label="anchor" href="#header"></a>
</h3>
<pre><code><span><span class="co">### PGS CATALOG SCORING FILE - see www.pgscatalog.org/downloads/#dl_ftp for additional information</span></span>
<span><span class="co">## POLYGENIC SCORE (PGS) INFORMATION</span></span>
<span><span class="co"># PGS ID = &lt;PGS idenfier, e.g. 'PGS000001'&gt;</span></span>
<span><span class="co"># PGS Name = &lt;PGS name, e.g. 'PRS77_BC' - optional&gt;</span></span>
<span><span class="co"># Reported Trait = &lt;trait, e.g. 'Breast Cancer'&gt;</span></span>
<span><span class="co"># Original Genome Build = &lt;Genome build/assembly, e.g. 'GRCh38'&gt;</span></span>
<span><span class="co"># Number of Variants = &lt;Number of variants listed in the PGS&gt;</span></span>
<span><span class="co">## SOURCE INFORMATION</span></span>
<span><span class="co"># PGP ID = &lt;PGS publication idenfier, e.g. 'PGP000001'&gt;</span></span>
<span><span class="co"># Citation = &lt;Information about the publication&gt;</span></span>
<span><span class="co"># LICENSE = &lt;License and terms of PGS use/distribution - refers to the EMBL-EBI Terms of Use by default&gt;</span></span></code></pre>
</div>
<div class="section level3">
<h3 id="columns">Columns<a class="anchor" aria-label="anchor" href="#columns"></a>
</h3>
<p>The following table lists all possible columns in a PGS scoring file.
A few columns are required, and most are optional; either the
<code>rsID</code> alone or the combination of <code>chr_name</code> and
<code>chr_position</code> are required, with the other being optional.
The <code><a href="../reference/read_scoring_file.html">read_scoring_file()</a></code> function automatically reads all
available columns with the right R types into a data frame. Original
source is <a href="https://www.pgscatalog.org/downloads/#scoring_columns" class="external-link uri">https://www.pgscatalog.org/downloads/#scoring_columns</a>.</p>
<table class="table">
<colgroup>
<col width="12%">
<col width="22%">
<col width="32%">
<col width="32%">
</colgroup>
<thead><tr class="header">
<th>Column</th>
<th>Field Name</th>
<th>Field Description</th>
<th>Field Requirement</th>
</tr></thead>
<tbody>
<tr class="odd">
<td><code>rsID</code></td>
<td>dbSNP Accession ID (rsID)</td>
<td>The SNP’s rs ID</td>
<td>
<strong>Optional</strong> – unless both the <em>chr_name</em> and
<em>chr_position</em> columns are absent.</td>
</tr>
<tr class="even">
<td><code>chr_name</code></td>
<td>Location - Chromosome</td>
<td>Chromosome name/number associated with the variant</td>
<td>
<strong>Required</strong> – may be optional if an <em>rsID</em> for
the variant is provided.</td>
</tr>
<tr class="odd">
<td><code>chr_position</code></td>
<td>Location - Position within the Chromosome</td>
<td>Chromosomal position associated with the variant</td>
<td>
<strong>Required</strong> – may be optional if an <em>rsID</em> for
the variant is provided.</td>
</tr>
<tr class="even">
<td><code>effect_allele</code></td>
<td>Effect Allele</td>
<td>The allele that’s dosage is counted (e.g. {0, 1, 2}) and multiplied
by the variant’s weight (‘effect_weight’) when calculating score. The
effect allele is also known as the ‘risk allele’.</td>
<td><strong>Required</strong></td>
</tr>
<tr class="odd">
<td><code>other_allele</code></td>
<td>Other Allele</td>
<td>The other allele(s) at the loci</td>
<td>Optional – but strongly recommended.</td>
</tr>
<tr class="even">
<td><code>effect_weight</code></td>
<td>Variant Weight</td>
<td>Value of the effect that is multiplied by the dosage of the effect
allele (‘effect_allele’) when calculating the score.</td>
<td><strong>Required</strong></td>
</tr>
<tr class="odd">
<td><code>locus_name</code></td>
<td>Locus Name</td>
<td>This is kept in for loci where the variant may be referenced by the
gene (APOE e4). It is also common (usually in smaller PGS) to see the
variants named according to the genes they impact.</td>
<td>Optional</td>
</tr>
<tr class="even">
<td><code>allelefrequency_effect</code></td>
<td>Effect Allele Frequency</td>
<td>Reported effect allele frequency, if the associated locus is a
haplotype then haplotype frequency will be extracted.</td>
<td>Optional</td>
</tr>
<tr class="odd">
<td><code>is_interaction</code></td>
<td>FLAG: Interaction</td>
<td>This is a TRUE/FALSE variable that flags whether the weight should
be multiplied with the dosage of more than one variant. Interactions are
demarcated with a <em><em>x</em></em> between entries for each of the
variants present in the interaction.</td>
<td>Optional</td>
</tr>
<tr class="even">
<td><code>is_recessive</code></td>
<td>FLAG: Recessive Inheritance Model</td>
<td>This is a TRUE/FALSE variable that flags whether the weight should
be added to the PGS sum only if there are 2 copies of the effect allele
(e.g. it is a recessive allele).</td>
<td>Optional</td>
</tr>
<tr class="odd">
<td><code>is_haplotype</code></td>
<td>FLAG: Haplotype</td>
<td>This is a TRUE/FALSE variable that flags whether the effect allele
is a haplotype/diplotype rather than a single SNP. Constituent SNPs in
the haplotype are semi-colon separated.</td>
<td>Optional</td>
</tr>
<tr class="even">
<td><code>is_diplotype</code></td>
<td>FLAG: Diplotype</td>
<td>This is a TRUE/FALSE variable that flags whether the effect allele
is a haplotype/diplotype rather than a single SNP. Constituent SNPs in
the haplotype are semi-colon separated.</td>
<td>Optional</td>
</tr>
<tr class="odd">
<td><code>imputation_method</code></td>
<td>Imputation Method</td>
<td>This described whether the variant was specifically called with a
specific imputation or variant calling method. This is mostly kept to
describe HLA-genotyping methods (e.g. flag SNP2HLA, HLA*IMP) that gives
alleles that are not referenced by genomic position.</td>
<td>Optional</td>
</tr>
<tr class="even">
<td><code>variant_description</code></td>
<td>Variant Description</td>
<td>This field describes any extra information about the variant
(e.g. how it is genotyped or scored) that cannot be captured by the
other fields.</td>
<td>Optional</td>
</tr>
<tr class="odd">
<td><code>inclusion_criteria</code></td>
<td>Score Inclusion Criteria</td>
<td>Explanation of when this variant gets included into the PGS (e.g. if
it depends on the results from other variants).</td>
<td>Optional</td>
</tr>
<tr class="even">
<td><code>OR</code></td>
<td>Odds Ratio</td>
<td>Sometimes the authors only give an Odds Ratio (OR) as a weight for
the PGS. In these cases the OR is turned into an
<code>effect_weight</code>, by taking the log, as is customary when
deriving PGS. The OR column in the scoring file to reflect how the score
was originally provided by the authors.</td>
<td>Optional</td>
</tr>
<tr class="odd">
<td><code>HR</code></td>
<td>Hazard Ratio</td>
<td>Sometimes the authors only give an Hazard Ratio (HR) as a weight for
the PGS. In these cases the HR is turned into an
<code>effect_weight</code>, by taking the log, as is customary when
deriving PGS. The HR column in the scoring file to reflect how the score
was originally provided by the authors.</td>
<td>Optional</td>
</tr>
<tr class="even">
<td>
<code>dosage_0_weight</code>, <code>dosage_1_weight</code>,
<code>dosage_2_weight</code>
</td>
<td>Effect weight with 0, 1 or 2 copies of the effect allele</td>
<td>Weights that are specific to different dosages of the effect_allele
(e.g. {0, 1, 2} copies) can also be reported when the the contribution
of the variants to the score is not encoded as additive, dominant, or
recessive. In this case three columns are added corresponding to which
variant weight should be applied for each dosage, where the column name
is formated as dosage_#_weight where the # sign indicates the number of
effect_allele copies.</td>
<td>Optional</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="section level2">
<h2 id="references">References<a class="anchor" aria-label="anchor" href="#references"></a>
</h2>
<p>The information here provided is gathered from the PGS Catalog
documentation on the PGS scoring file schema: <a href="https://www.pgscatalog.org/downloads/" class="external-link">PGS Scoring Files</a>.</p>
</div>
  </main><aside class="col-md-3"><nav id="toc" aria-label="Table of contents"><h2>On this page</h2>
    </nav></aside>
</div>



    <footer><div class="pkgdown-footer-left">
  <p>Developed by <a href="https://www.pattern.institute/team/rmagno/" class="external-link">Ramiro Magno</a>, <a href="http://iduarte.eu/" class="external-link">Isabel Duarte</a>, Ana-Teresa Maia, <a href="https://cintesis.eu/" class="external-link"><img src="https://rmagno.eu/quincunx/reference/figures/cintesis.svg" height="24" alt="CINTESIS logo"></a>, <a href="https://www.pattern.institute/" class="external-link"><img src="https://rmagno.eu/quincunx/reference/figures/pi-logotype-canonical-whole-v01.svg" height="24" alt="Pattern Institute logo"></a>.</p>
</div>

<div class="pkgdown-footer-right">

</div>

    </footer>
</div>





  </body>
</html>
